packages/playground/windows/playground-win32/Playground-Win32.cpp

```diff
@@ -14,35 +14,46 @@
 #pragma push_macro("GetCurrentTime")
 #undef GetCurrentTime
 
+#include <DesktopWindowBridge.h>
 #include <winrt/Microsoft.ReactNative.h>
 
 #include <UI.Xaml.Controls.h>
 #include <UI.Xaml.Hosting.h>
 #include <winrt/Windows.Foundation.Collections.h>
-
 #pragma pop_macro("GetCurrentTime")
 
 #ifndef USE_WINUI3
 namespace xaml = winrt::Windows::UI::Xaml;
 #else
+#include <winrt/Microsoft.UI.h>
 namespace xaml = winrt::Microsoft::UI::Xaml;
 #endif
 
 namespace controls = xaml::Controls;
 namespace hosting = xaml::Hosting;
 
+#define WINDOWDATA L"WindowData"
+
 int RunPlayground(int showCmd, bool useWebDebugger);
 
+HWND GetXamlIslandHwnd(const hosting::DesktopWindowXamlSource &dwxs) {
+  auto interop = dwxs.as<IDesktopWindowXamlSourceNative>();
+  // Get the new child window's hwnd
+  HWND hWndXamlIsland = nullptr;
```

Comment:
>nullptr [](start = 24, length = 7)

We usually use NULL for the empty handle.

Reply:
we should be using nullptr for HWND (or INVALID_HANDLE for NT handles)  - (Windows guidelines)